library/std/src/net/ip.rs

```diff
@@ -1436,18 +1700,28 @@ impl Ipv6Addr {
     /// );
     /// assert_eq!(Ipv6Addr::new(0, 0, 0, 0, 0, 0xffff, 0xc00a, 0x2ff).multicast_scope(), None);
     /// ```
+    ///
+    /// # Stability guarantees
+    ///
+    /// This method's behavior [may by subject to changes][changes] in the
+    /// future, as new RFCs are published.
+    ///
+    /// [changes]: ../net/index.html#stability-guarantees-for-ietf-defined-behavior
+    #[stable(feature = "ip", since = "1.42.0")]
     pub fn multicast_scope(&self) -> Option<Ipv6MulticastScope> {
+        use Ipv6MulticastScope::*;
```

Comment:
I think we should keep the `Ipv6MulticastScope` to be specific, I don't know if the `*` import would be good here.